src/simmate/apps/clease/workflows/cluster_high_q.py

```diff
@@ -1,10 +1,45 @@
-from simmate.apps.vasp.inputs import PBE_POTCAR_MAPPINGS
+from simmate.apps.vasp.inputs.potcar_mappings import PBE_ELEMENT_MAPPINGS
 from simmate.apps.vasp.workflows.base import VaspWorkflow
-
+from simmate.apps.vasp.error_handlers import (
+    Brions,
+    Brmix,
+    Edddav,
+    Eddrmm,
+    Edwav,
+    ElfKpar,
+    Frozen,
+    IncorrectShift,
+    IncorrectSmearing,
+    InsufficientBands,
+    LargeSigma,
+    LongVector,
+    MeshSymmetry,
+    NonConverging,
+    PointGroup,
+    PositiveEnergy,
+    Posmap,
+    Potim,
+    Pricel,
+    Pssyevx,
+    RealOptlay,
+    Rhosyg,
+    RotationMatrix,
+    RotationNonIntMatrix,
+    SubspaceMatrix,
+    SymprecNoise,
+    Tetirr,
+    TetrahedronMesh,
+    TripleProduct,
+    Unconverged,
+    Walltime,
+    Zbrent,
+    Zheev,
+    Zpotrf,
+)
```

Comment:
I'll make a "common error handlers" utility so we don't have to write all of these out every time